helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -697,8 +693,7 @@ public void reset() {
   }
 
   void reset(boolean isShutdown) {
-    logger.info("Resetting CallbackHandler: {}. Is resetting for shutdown: {}.", this.toString(),
-        isShutdown);
+    logger.info("Reset CallbackHandler. isShutdown: {}. Content: {}", isShutdown, getContent());
```

Comment:
Same here, callback type is necessary.

Reply:
The callback type is not passed in from the caller in the beginning. Do you want to pass that information? And getContent gives you all the information

Reply:
getContent should be good enough. Good point.